### ISSUE TITLE
Nagstamon-3.13-20230921 is broken with Centreon 22.10 (at least)

### ISSUE THREAD

**Issue by BenoitPoulet** [HenriWahl/Nagstamon#966]
Hello,

A team mate installed the latest build and he is unable to load the status.

The problem was not present in the 3.10.

```
DEBUG: 2023-09-21 15:00:13.523119 Saving configuration to file c:\users\benoit.poulet\.nagstamon\nagstamon.conf
DEBUG: 2023-09-21 15:00:24.326904 Centreon Dev Created server.
DEBUG: 2023-09-21 15:00:24.326904 Centreon Pre Prod Created server.
DEBUG: 2023-09-21 15:00:24.326904 Centreon Prod FetchURL: https://centreon/centreon/api/latest/platform/versions CGI Data: None
DEBUG: 2023-09-21 15:00:24.549263 [Centreon Prod] Status code 200
DEBUG: 2023-09-21 15:00:24.549263 [Centreon Prod] Loading class API, Centreon version : 22.10
DEBUG: 2023-09-21 15:00:24.549263 Centreon Prod Created server.
DEBUG: 2023-09-21 15:00:24.549263 Demo Centreon User Created server.
DEBUG: 2023-09-21 15:00:24.549263 Demo Centreon admin Created server.
DEBUG: Initializing SystemTrayIcon
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD1FC0> for state "OK"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD1EA0> for state "INFORMATION"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD2050> for state "UNKNOWN"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD20E0> for state "WARNING"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD2170> for state "AVERAGE"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD2200> for state "HIGH"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD2290> for state "CRITICAL"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD2320> for state "DISASTER"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD23B0> for state "UNREACHABLE"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD2440> for state "DOWN"
DEBUG: SystemTrayIcon created icon <Nagstamon.QUI.QIconWithFilename object at 0x000001BEB0FD24D0> for state "ERROR"
DEBUG: SystemTrayIcon initial icon: <PyQt6.QtGui.QIcon object at 0x000001BEB0FD1CF0>
DEBUG: 2023-09-21 15:00:26.344465 Centreon Prod FetchURL: https://centreon/centreon/api/latest/platform/versions CGI Data: None
DEBUG: 2023-09-21 15:00:26.611978 [Centreon Prod] Checking session status
DEBUG: 2023-09-21 15:00:26.611978 [Centreon Prod] Check-session, the token expire if not been used for more than one hour. Current Token = None
ERROR: 2023-09-21 15:00:26.614061 Centreon Prod Traceback (most recent call last):
  File "C:\Users\benoit.poulet\Dépots\GitHub\Nagstamon\Nagstamon\Servers\Centreon\CentreonAPI.py", line 734, in check_session
    result = self.FetchURL(self.urls_centreon['resources'] + '?' + urllib.parse.urlencode(cgi_data),
                           ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
```

**Comment by HenriWahl** [comment]
Strange - will you be able to find out why self.urls_centreon is of type NoneType?

**Comment by BenoitPoulet** [comment]
I found that 
    `def init_config(self):`
is not called anymore.

I added 
```
        self.Debug(server='[' + self.get_name() + ']',
                   debug='>>>>>> init_config')
```

at the end, and it never happens

**Comment by BenoitPoulet** [comment]
Hello,

I found the cause, some functions are not returning the same value as before.
I'm working on a fix.

**Comment by HenriWahl** [comment]
Available as 3.13-20230927 now.
